package.json

```diff
@@ -23,36 +20,36 @@
   },
   "devDependencies": {
     "@springernature/eslint-config": "^2.0.0",
-    "eslint": "^5.1.0",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-jest": "^21.17.0",
+    "eslint": "~5.12.0",
+    "eslint-plugin-import": "~2.14.0",
+    "eslint-plugin-jest": "~21.27.2",
     "eslint-plugin-no-use-extend-native": "^0.3.12",
     "eslint-plugin-node": "^6.0.1",
     "eslint-plugin-promise": "^3.8.0",
     "eslint-plugin-unicorn": "^4.0.3",
-    "jest": "^23.1.0",
-    "mock-fs": "^4.5.0",
-    "nock": "^9.6.0",
+    "jest": "~23.6.0",
+    "mock-fs": "~4.7.0",
+    "nock": "~9.6.1",
     "pmock": "^0.2.3",
     "q": "^1.5.0",
     "rewire": "^3.0.2",
     "strip-ansi": "^4.0.0"
   },
   "dependencies": {
-    "chalk": "^2.4.1",
-    "fs-extra": "^7.0.0",
+    "chalk": "~2.4.2",
+    "fs-extra": "~7.0.1",
     "gitignore-globs": "^0.1.1",
-    "glob": "^7.1.2",
-    "inquirer": "^6.1.0",
-    "listr": "^0.14.1",
+    "glob": "~7.1.3",
+    "inquirer": "~6.2.1",
+    "listr": "~0.14.3",
     "minimatch": "^3.0.4",
     "npm-registry-client": "^8.3.0",
     "npm-utils": "^2.0.3",
     "pretty-error": "^2.1.1",
     "sanitize-filename": "^1.6.1",
-    "semver": "^5.3.0",
+    "semver": "~5.6.0",
     "validate-npm-package-name": "^3.0.0",
-    "yargs": "^12.0.1"
+    "yargs": "~12.0.5"
   },
   "jest": {
     "testURL": "http://localhost/",
```

Comment:
we've got a combination of `^` and `~` in the deps now. Should this be consistent? As a team did we have a consensus on what to use?

Reply:
nice catch, and yep - tildes. Not sure why `npm update` did that, will fix